// @slidev/parser/dist/index.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});
















var _chunkU5Q7HAO4js = require('./chunk-U5Q7HAO4.js');

















exports.detectFeatures = _chunkU5Q7HAO4js.detectFeatures; exports.filterDisabled = _chunkU5Q7HAO4js.filterDisabled; exports.getDefaultConfig = _chunkU5Q7HAO4js.getDefaultConfig; exports.mergeFeatureFlags = _chunkU5Q7HAO4js.mergeFeatureFlags; exports.parse = _chunkU5Q7HAO4js.parse; exports.parseAspectRatio = _chunkU5Q7HAO4js.parseAspectRatio; exports.parseRangeString = _chunkU5Q7HAO4js.parseRangeString; exports.parseSlide = _chunkU5Q7HAO4js.parseSlide; exports.prettify = _chunkU5Q7HAO4js.prettify; exports.prettifySlide = _chunkU5Q7HAO4js.prettifySlide; exports.resolveConfig = _chunkU5Q7HAO4js.resolveConfig; exports.resolveFonts = _chunkU5Q7HAO4js.resolveFonts; exports.scanMonacoModules = _chunkU5Q7HAO4js.scanMonacoModules; exports.stringify = _chunkU5Q7HAO4js.stringify; exports.stringifySlide = _chunkU5Q7HAO4js.stringifySlide; exports.verifyConfig = _chunkU5Q7HAO4js.verifyConfig;
